### DISCUSSION TITLE
Better color tools: color editor in the main window.

### DISCUSSION THREAD

**Discussion by TheGoklayeh** [Orama-Interactive/Pixelorama#573] [Ideas]
We need to be able to easily edit colors in the main panel. With the different value systems (RGB, HSV/B, HSL). It doen't have to be big nor complex, something like this would be enough: 
![hjtcdtyec](https://user-images.githubusercontent.com/38734968/78561506-23006c80-7818-11ea-889c-8a638c93e9a9.PNG)
This would allow to easily modify colors, use them, and add then to the current palette.

**Comment by Calinou** [comment]
> Other useful things:

Please create one issue per feature request after searching for possible duplicates. This makes it easier to check off individual features once they've been implemented.

**Comment by TheGoklayeh** [comment]
> Please create one issue per feature request.

Hi, I'll change that, just wanted to avoid spamming the place.

**Comment by OverloadedOrama** [comment]
This is implemented in #953.
